### PR TITLE
add redirect url to quicket for tickets

### PR DESCRIPTION
**Pull request by basiima**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -3,6 +3,15 @@ const nextConfig = {
   experimental: {
     appDir: true,
   },
+  async redirects() {
+    return [
+      {
+        source: '/tickets',
+        destination: 'https://qkt.io/w31vDE',
```
Comment (HassanBahati) [inline next.config.js:10]: How possible is it read this value from environment variables?

Comment (HassanBahati) [reply next.config.js:10]: Would be nice to make it easy to delete code on tickets...if it's from a central place

Comment (Amanya-Brian) [reply next.config.js:10]: I agree with you @HassanBahati. I think that will be it

**Review by Amanya-Brian — APPROVED** [review]
Had just worked on this too. Well done!

File: next.config.js (hunk 1 above)
Comment (basiima) [reply next.config.js:10]: what do you mean by deleting code on tickets?

Comment (HassanBahati) [reply next.config.js:10]: Deleting code...I mean...if incase there is a change....there shouldnt so many places to change...placing the actual link in so many places means all those will need to change.....best to have the actual link in one place and reference it by variables in all these places

Comment (HassanBahati) [reply next.config.js:10]: That's why was suggesting to have it as a variable

**Pull request merged**